.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Keeping it in repository for easy debug 
```

Comment:
You should NOT check in the launch.json -- you should add it to .gitignore and keep your local copy